### PR TITLE
Implement the logic to store the forms' content

### PR DESCRIPTION
**Pull request by Estete9**
Functionality that saves the name, email, and message into the localStorage
The form is repopulated when the page loads
The form resets when submitted successfully

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @Team :wave:  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with failing linter checks. Kindly fix the Eslint error

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -175,6 +175,36 @@ form.addEventListener('submit', (event) => {
...
+}
+
+// loads from local storage and updates the form's content
+function repopulateForm() {
+  for (let i = 0; i < elementsArr.length; i += 1) {
+    elementsArr[i].value = getFromStorage(elementsArr[i].id);
+  }
+}
+
+window.onload = repopulateForm();
```
Comment (Owusu-Desmond) [inline script.js]: - [ ] Please ensure your contact form information is persistent to page reloads. After filling the form, and not submitting it, the entered values should be saved in the browser. Also when visiting the page the form inputs should be contained, and auto-populated with, the values entered in the previous visit to the page. Currently, your form is not preserving the form data in the localStorage of the browser so make sure that you fix this issue.
  
  [Screencast from 07-28-2023 07:58:26 PM.webm](https://github.com/Estete9/Fullstack-Portfolio/assets/92303891/0d3c043a-988c-471e-b390-503edda2bd41)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Estete9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Estete9) [reply script.js]: Hey! I'm very grateful for your feedback. 
Can you show me how to reproduce the error? As you can see in this video, I did the same as in yours but in mine, the information was preserved.
https://www.loom.com/share/ec13d4aae94d4db18ad47e0c1eefb970

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Estete9 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Form data persistent to reloads 📌 
✔️ Linters are passing 🟢 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**